lor-init

```diff
@@ -115,7 +115,7 @@ function lor-config() {
 	# Enable ansible_mitogen strategy plugin for improved speed of plays when available.
 	#
 	_ansible_mitogen_search_path='.*/lib/python.*/site-packages/ansible_mitogen/plugins/strategy'
-	_ansible_mitogen_path=$(find ${VIRTUAL_ENV:-.} -regex "${_ansible_mitogen_search_path}" | sort -V | tail -1)
+	_ansible_mitogen_path=$(find ${VIRTUAL_ENV:-.}/lib -type d -name mitogen | sort -V | tail -1)
```

Comment:
Does this work? It does not the same path to to `ANSIBLE_STRATEGY_PLUGINS`;
`league-of-robots/python.venv/lib/python*/site-packages/mitogen/`
is not the same as
`league-of-robots/python.venv/lib/python*/site-packages/ansible_mitogen/plugins/strategy/`

Reply:
1) you don't need to apply regex to all output (which is why this is time consuming), instead you just look within a `python.venv/`, which was already detected before, and under `lib` subdirectory (which holds various python environments) look for a folder `mitogen`
2) it works on linux but I don't think of a reason why would be find command system dependent ... one can never be sure on mac
3) indeed `_ansible_mitogen_search_path` is not needed anymore - I removed it with new commit

Reply:
I get that, but the logic has changed and you are now adding a completely different path to ANSIBLE_STRATEGY_PLUGINS: I'd be surprised if the requested `mitogen_linear` strategy still works. More likely it will fail to find that and default back to strategy `linear`.

Reply:
of course - I simplified too much and changed original file too much ... and while I did so, I ended up searching for `mitogen` instead `stategy` :D

Reply:
I am not sure how much in `python.venv/lib/` folder the name `pythonX.XX` is changing, but new update is searching for `python*`, which could be further changed even in `*`. But all in all this works and is fast.